plans/rust-keylime-tests.fmf

```diff
@@ -3,6 +3,7 @@ summary:
 
 environment+:
   TPM_BINARY_MEASUREMENTS: /var/tmp/binary_bios_measurements
+  KEYLIME_RUST_CODE_COVERAGE: 1
```

Comment:
We should make it conditional as for Python and measure coverage only on Fedora37 and CS9.